__tests__/addaccount.test.js

```diff
@@ -36,7 +40,47 @@ test('When a view only address is added should be displayed in dashboard metrics
   await clearLocalStorage(FIXTURE_MYC_STORAGE_KEY);
   await addAccountPage.addViewOnly();
 
-  dashboardPage.waitPageLoaded();
+  await dashboardPage.waitPageLoaded();
 
   FIXTURE_VIEW_ONLY_TOKENS.forEach((t) => dashboardPage.expectBalanceInBalanceList(t));
 });
+
+// Add Account - Private Key
+test('Should be able to add a private key address', async (t) => {
+  await clearLocalStorage(FIXTURE_MYC_STORAGE_KEY);
+  await addAccountPage.addPrivateKey();
+
+  await dashboardPage.waitPageLoaded();
+
+  await dashboardPage.expectAddressToBePresent(FIXTURE_PRIVATE_KEY_ADDRESS);
+  await dashboardPage.expectAccountTableToMatchCount(1);
+});
+
+test('When a private key address is added should be displayed in dashboard metrics', async (t) => {
+  await clearLocalStorage(FIXTURE_MYC_STORAGE_KEY);
+  await addAccountPage.addPrivateKey();
+
+  await dashboardPage.waitPageLoaded();
+
+  FIXTURE_PRIVATE_KEY_TOKENS.forEach((t) => dashboardPage.expectBalanceInBalanceList(t));
+});
+
+// Add Account - Keystore File
+test('Should be able to add a keystore file address', async (t) => {
+  await clearLocalStorage(FIXTURE_MYC_STORAGE_KEY);
+  await addAccountPage.addKeystoreFile();
+
+  await dashboardPage.waitPageLoaded(FIXTURES_CONST.TIMEOUT * 2);
+
+  await dashboardPage.expectAddressToBePresent(FIXTURE_PRIVATE_KEY_ADDRESS);
+  await dashboardPage.expectAccountTableToMatchCount(1);
+});
+
+test('When a keystore file address is added should be displayed in dashboard metrics', async (t) => {
+  await clearLocalStorage(FIXTURE_MYC_STORAGE_KEY);
+  await addAccountPage.addKeystoreFile();
+
+  await dashboardPage.waitPageLoaded(FIXTURES_CONST.TIMEOUT * 2);
+
+  FIXTURE_PRIVATE_KEY_TOKENS.forEach((t) => dashboardPage.expectBalanceInBalanceList(t));
+});
```

Comment:
Each page load is reeeeally expensive since it reloads the entire app.
Prefer to write fewer and longer tests, where you follow the app flow and chain expectations.
We may need to remove the expectations from the dashboard page.

Reply:
I made a change to not delete them but to combine them into one test so we only need to load the dashboard once per wallet type instead of 2x. If we notice that being slow in the future, we can work something else out